projects/packages/forms/src/dashboard/landing/index.js

```diff
@@ -0,0 +1,204 @@
+import { createInterpolateElement } from '@wordpress/element';
```

Comment:
Should we just turn that into a 'view'? The way I imagined it initially `components/` were meant to be reusable and then we'd create 'top-level directories' for everything else.

So, shall we move it to `/dashboard/landing-page` ?

Reply:
Not sure if understood what you mean.
The landing page is already under `/dashboard`, but it's called `landing`.
Are you suggesting renaming it to `landing-page`?